packages/flutter/lib/src/widgets/app.dart

```diff
@@ -681,6 +684,49 @@ class WidgetsApp extends StatefulWidget {
   /// {@endtemplate}
   final bool debugShowCheckedModeBanner;
 
+  /// {@template flutter.widgets.widgetsApp.shortcuts}
+  /// The default map of keyboard shortcuts to intents for the application.
+  ///
+  /// By default, this is the output of [WidgetsApp.defaultShortcuts], called
+  /// with [defaultTargetPlatform]. If you wish to modify the default
```

Comment:
This sounds rather complicated. Maybe show this in an example?

(Also, since you're not doing it in the test, I assume it's also ok to just provide a new map without calling defaultShortcuts?)